scripts/start.sh

```diff
@@ -0,0 +1,396 @@
+    "tls_mode": "Auto"
+  }
+}
+EOF
+)" >/dev/null
+
+# Wait until NATS system is connected to control plane
+checkSystemState() {
+  SYSTEM_STATE=$(request GET "/systems/$SYSTEM_ID" | jq --raw-output .state)
+  if [ "$SYSTEM_STATE" != 'Connected' ]; then
+    exit 1
+  fi
+}
+retry_with_backoff checkSystemState || (rt=$?; red 'System failed to connect to NATS server' >&2; exit $rt)
+
+# === Create a user and account ===
+ACCOUNT_ID=$(request POST "/systems/$SYSTEM_ID/accounts" --data '{"name":"trial"}' | jq --raw-output .id)
+
+SK_GROUP_ID=$(request GET "/accounts/${ACCOUNT_ID}/account-sk-groups/" | jq --raw-output '.items[0].id')
+
+NATS_USER_ID=$(request POST "/accounts/${ACCOUNT_ID}/nats-users/" \
+  --data "$(cat <<EOF | jq --compact-output
+{
+  "name": "trial",
+  "sk_group_id": "${SK_GROUP_ID}",
+  "jwt_expires_in_secs": 0
+}
+EOF
+)" | jq --raw-output .id)
+
+request POST "/nats-users/${NATS_USER_ID}/creds" > trial.creds
+
+# === Setup HTTP Gateway ===
+
+# === Create an account, user, and KV bucket ===
+HTTP_GATEWAY_ACCOUNT_ID=$(request POST "/systems/$SYSTEM_ID/accounts" --data '{"name":"http-gateway"}' | jq --raw-output .id)
+
+HTTP_GATEWAY_SK_GROUP_ID=$(request GET "/accounts/${HTTP_GATEWAY_ACCOUNT_ID}/account-sk-groups/" | jq --raw-output '.items[0].id')
+
+HTTP_GATEWAY_NATS_USER_ID=$(request POST "/accounts/${HTTP_GATEWAY_ACCOUNT_ID}/nats-users/" \
+  --data "$(cat <<EOF | jq --compact-output
+{
+  "name": "http-gateway",
+  "sk_group_id": "${HTTP_GATEWAY_SK_GROUP_ID}",
+  "jwt_expires_in_secs": 0
+}
+EOF
+)" | jq --raw-output .id)
+
+request POST "/nats-users/${HTTP_GATEWAY_NATS_USER_ID}/creds" > http-gateway.creds
+
+createKVBucket() {
+  request POST "/accounts/${HTTP_GATEWAY_ACCOUNT_ID}/jetstream/kv-buckets/" --data '{"bucket":"tokens"}' >/dev/null
+}
+retry_with_backoff "createKVBucket" || (rt=$?; red 'Failed to create KV bucket for HTTP Gateway account' >&2; exit $rt)
+
+request PATCH "/systems/$SYSTEM_ID/" \
+  --data "$(cat <<EOF | jq --compact-output
+{
+  "http_gateway_config": {
+    "account": "${HTTP_GATEWAY_ACCOUNT_ID}",
+    "token_bucket": "tokens",
+    "url": "http://localhost:8081"
+  }
+}
+EOF
+)" >/dev/null
+
+docker compose up --detach --wait http-gateway
+
+HTTP_GATEWAY_TOKEN=$(request POST "/nats-users/${HTTP_GATEWAY_NATS_USER_ID}/http-gw-token" | jq --raw-output .token)
+echo "HTTP_GATEWAY_TOKEN=\"${HTTP_GATEWAY_TOKEN}\"" >> .env
+bold '\nSaved HTTP_GATEWAY_TOKEN to .env\n'
+
+echo -e "Done bootstrapping Synadia Platform, open the UI at $(link 'http://localhost:8080' 'http://localhost:8080') and log in with:\n\n    username: $(bold 'admin')\n    password: $(bold "$ADMIN_PASSWORD")\n"
+
+# if --open, open the browser to control plane
+if [ -n "$open" ]; then
+  case "$(uname -s)" in
+    Linux)
+      if check_command xdg-open; then
+        xdg-open "http://localhost:8080"
+      fi
+      ;;
+    Darwin)
+      open "http://localhost:8080"
+      ;;
+  esac
+fi
+
+copy_to_clipboard "$ADMIN_PASSWORD"
```

Comment:
I wouldn't copy to clipboard without asking :-)

Reply:
Good call, done: https://github.com/synadia-io/platform-trial/pull/1/commits/53b0321aae47dff15c905a7a5e0db794824efeb2